cmd/opm/alpha/template/composite.go

```diff
@@ -49,6 +50,8 @@ and a 'composite template' file`,
 			}
 			defer compositeReader.Close()
 
+			compositePath, _ := filepath.Abs(filepath.Dir(compositeFile))
```

Comment:
Looks like we are eating an error here. Is there a particular reason for this?

Looking into the `filepath.Abs()` source code this returns an error if `os.Getwd()` returns an error (at least on unix based systems).

Reply:
My first thoughts were that any access error would be caught in line 49 where the same file is opened.  However, it's not really a big deal if we catch the error.  I just can't imagine a case where `Abs` generates an error that `Open` would not.

Reply:
Yeah, that's fair. I just personally prefer to not eat errors because it can cause a lot of issues if we assume something that has the potential to error will never error